src/Fantomas.Core/ASTTransformer.fs

```diff
@@ -966,22 +966,22 @@ let mkExpr (creationAide: CreationAide) (e: SynExpr) : Expr =
     | SynExpr.AddressOf(false, e, _, StartRange 2 (ampersandToken, _range)) ->
         ExprSingleNode(stn "&&" ampersandToken, false, false, mkExpr creationAide e, exprRange)
         |> Expr.Single
-    | SynExpr.YieldOrReturn((true, _), e, StartRange 5 (yieldKeyword, _range)) ->
+    | SynExpr.YieldOrReturn((true, _), e, StartRange 5 (yieldKeyword, _range), _) ->
```

Comment:
We should not ignore the trivia here, replace `yieldKeyword` with this instead: https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-syntaxtrivia-synexpryieldorreturntrivia.html

Reply:
Mmh. I guess I know too little about ASTs and Oaks. But isn't `yieldKeyword` just the range of the `"yield"`? It looks right to me as is. While `SynExprYieldOrReturnTrivia` is the type of the trivia-range-wrapper that I ignored. But what should I do with this range?

Reply:
What we do with `StartRange 5 (yieldKeyword, _range))` is creating a new range for the `yield` keyword using the range of the entire SynExpr.
But since the trivia, now hold the same information, we can ignore the range and use the trivia to get the `yieldKeyword` range instead.